unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/SceneController.cs

```diff
@@ -761,15 +766,14 @@ public void UnloadAllScenesQueued()
             Environment.i.messaging.manager.ForceEnqueueToGlobal(MessagingBusType.INIT, queuedMessage);
         }
 
-        public void CreateGlobalScene(string json)
+        public void CreateGlobalScene(CreateGlobalSceneMessage globalScene)
```

Comment:
Won't this break the current flow? am I missing something?

Reply:
When that change was introduced, the `SceneControllerBridge` was also updated to cope with that change and keep everyrthing working, check the bridge changes at [the corresponding commit](https://github.com/decentraland/unity-renderer/pull/3700/commits/90326c570c71d713e18eeaf2447680c2f477eaee)